### PR TITLE
W-11077803 salesforce marketing cloud reference cleanup

### PR DESCRIPTION
**Pull request by sathya0**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc [hunk 1]
```diff
@@ -53,25 +48,25 @@ Release Notes: xref:release-notes::connector/salesforce-mktg-connector-release-n
...
+| Reconnection a| <<Reconnection>> | Configures a reconnection strategy to use when a connector operation fails to connect to an external server. |  |
 |===
-[[Config_OauthClientCredentials]]
-===== OAuth Client Credentials
 
+[[Config_OauthClientCredentials]]
+=== OAuth Client Credentials Connection Type
 
-====== Parameters
+Uses a client ID, a client secret, and a token URL for connection.
```
Comment (dhirsch1) [inline salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc]: Suggestion: Uses a client ID, client secret, and token URL for the connection.

File: salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc [hunk 2]
```diff
@@ -80,18 +75,109 @@ Release Notes: xref:release-notes::connector/salesforce-mktg-connector-release-n
...
+=== On new object
+`<sfdc-marketing-cloud:new-object-listener>`
+
+Listener for new objects.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Since a| String |  Specify a date in the `yyyy-MM-dd HH:mm:ss` format, for example, `2017-03-17 16:30:40`, to retrieve the selected objects. |  |
```
Comment (dhirsch1) [inline salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc]: Can you say what this field does?

File: salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc [hunk 3]
```diff
@@ -216,24 +302,22 @@ Creates one or more API objects. IMPORTANT: When you map your objects to the inp
...
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | x
+| Configuration | String | Name of the configuration to use. | | x
 | Object Type a| String |  Type of object to delete. |  | x
-| Api Objects a| Array of Object |  An array of one or more API objects . |  #[payload] |
-| DeleteOptions a| Object |  Options that can be made on the delete operation. |  |
-| Config Ref a| ConfigurationProvider |  The name of the configuration to use to execute this component. |  | x
-| Amount a| Number |  Specifies the amount of time, in the unit defined in Read timeout time unit, that the consumer waits for a response before it times out. |  |
+| Api Objects a| Array of Object |  Array of one or more API objects. |  `#[payload]` |
+| DeleteOptions a| Object |  Options for the operation. |  |
```
Comment (dhirsch1) [inline salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc:318]: Suggestion: Delete options..

File: salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc [hunk 4]
```diff
@@ -568,282 +652,279 @@ Upserts one or more API objects. IMPORTANT: When you map your objects to the inp
...
-[%header%autowidth.spread]
+Configures the repeatable in memory iterable type.
+
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Frequency a| Number | How often in milliseconds to reconnect. | |
-| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
+| Initial Buffer Size a| Number | Amount of instances to initially keep in memory to consume the stream and provide random access to it. If the stream contains more data than can fit into this buffer, then the buffer expands according to the *Buffer Size Increment* attribute, with an upper limit of *Max In Memory Size*. Default value is 100 instances. |  |
+| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of `0` or lower means that the buffer must not expand, meaning that a `STREAM_MAXIMUM_SIZE_EXCEEDED` error is raised when the buffer gets full. Default value is 100 instances. |  |
```
Comment (dhirsch1) [inline salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc]: In the spreadsheet:

Amount by which the buffer size expands if it exceeds its initial size. Setting a value of `0` or lower specifies that the buffer can't expand. 

You can add the next sentence. In terms of the default, I would add it in the last column and remove the text from here.

**Review by dhirsch1 — DISMISSED** [review]

File: salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc (hunk 3 above)
Comment (sathya0) [reply salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc:318]: This one is a UI name unfortunately :/

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**